src/main/java/org/fundacion/jala/converter/core/ExtractText.java

```diff
@@ -14,12 +14,11 @@
 import net.sourceforge.tess4j.TesseractException;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.fundacion.jala.converter.core.exceptions.TextExtractorException;
 import org.fundacion.jala.converter.core.parameter.ExtractTextParameter;
 import org.fundacion.jala.converter.core.results.Result;
-import java.io.BufferedWriter;
-import java.io.File;
-import java.io.FileWriter;
-import java.io.PrintWriter;
+
```

Comment:
The empty line between imports should be removed.

Reply:
Thanks, I have just fixed it